Add a test for 'should redirect to dashboard when logging in from /login'

Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [test/integration/login_flow_test.rb](https://github.com/publiclab/plots2/blob/main/test/integration/login_flow_test.rb) in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -64,6 +64,16 @@ class LoginFlowTest < ActionDispatch::IntegrationTest
     assert_equal '/questions', path
   end
 
+  test 'should redirect to dashboard when logging in from /login' do
+    get '/login'
+    assert_response :success
+
+    post '/user_sessions', params: { return_to: request.path, user_session: { username: users(:jeff).username,  password: 'secretive' }  }
+
+    follow_redirect!
+    assert_equal '/dashboard', path
+  end
+
   test 'google login routing' do
     assert_routing '/auth/google_oauth2/callback', {controller: 'user_sessions', action: 'create',provider: 'google_oauth2'}
   end
```


- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:
1. If you are not familiar with GitHub or the pull request model, [here is a guide you can follow](https://guides.github.com/activities/hello-world/) on how GitHub works.
2. If you are familiar with the terminal or would like to learn to use it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

3. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

Hmmmm, tests passed locally, but not when I submitted the PR. 
Looks like it might be an issue with CodeCov somehow? 

`bad argument (expected URI object or URI string)
/opt/hostedtoolcache/Ruby/2.6.6/x64/lib/ruby/2.6.0/uri/common.rb:739:in `URI'
/home/runner/work/plots2/plots2/vendor/bundle/ruby/2.6.0/gems/codecov-0.5.2/lib/codecov/uploader.rb:457:in `upload_to_v4'
/home/runner/work/plots2/plots2/vendor/bundle/ruby/2.6.0/gems/codecov-0.5.2/lib/codecov/uploader.rb:423:in `upload_to_codecov'
/home/runner/work/plots2/plots2/vendor/bundle/ruby/2.6.0/gems/codecov-0.5.2/lib/codecov/uploader.rb:43:in `upload'
/home/runner/work/plots2/plots2/vendor/bundle/ruby/2.6.0/gems/codecov-0.5.2/lib/codecov.rb:15:in `format'
/home/runner/work/plots2/plots2/vendor/bundle/ruby/2.6.0/gems/simplecov-0.21.2/lib/simplecov/result.rb:51:in `format!'
/home/runner/work/plots2/plots2/vendor/bundle/ruby/2.6.0/gems/simplecov-0.21.2/lib/simplecov/configuration.rb:197:in `block in at_exit'
/home/runner/work/plots2/plots2/vendor/bundle/ruby/2.6.0/gems/simplecov-0.21.2/lib/simplecov.rb:189:in `run_exit_tasks!'
/home/runner/work/plots2/plots2/vendor/bundle/ruby/2.6.0/gems/simplecov-0.21.2/lib/simplecov.rb:179:in `at_exit_behavior'
/home/runner/work/plots2/plots2/vendor/bundle/ruby/2.6.0/gems/simplecov-0.21.2/lib/minitest/simplecov_plugin.rb:11:in `block in plugin_simplecov_init'
/home/runner/work/plots2/plots2/vendor/bundle/ruby/2.6.0/gems/minitest-5.14.4/lib/minitest.rb:64:in `reverse_each'
/home/runner/work/plots2/plots2/vendor/bundle/ruby/2.6.0/gems/minitest-5.14.4/lib/minitest.rb:64:in `block (2 levels) in autorun'
Error: Process completed with exit code 1.`

--- a/test/integration/login_flow_test.rb
+++ b/test/integration/login_flow_test.rb
@@ -68,7 +68,7 @@ class LoginFlowTest < ActionDispatch::IntegrationTest
     get '/login'
     assert_response :success
 
-    post '/user_sessions', params: { return_to: request.path, user_session: { username: users(:jeff).username,  password: 'secretive' }  }
+    post '/user_sessions', params: { user_session: { username: users(:jeff).username,  password: 'secretive' }  }
 
     follow_redirect!
     assert_equal '/dashboard', path

--- a/test/integration/login_flow_test.rb
+++ b/test/integration/login_flow_test.rb
@@ -58,7 +58,7 @@ class LoginFlowTest < ActionDispatch::IntegrationTest
     get '/questions'
     assert_response :success
 
-    post '/user_sessions', params: { return_to: request.path, user_session: { username: users(:jeff).username,  password: 'secretive' }  }
+    post '/user_sessions', params: { return_to: request.path, user_session: { username: users(:jeff).username, password: 'secretive' }  }
 
     follow_redirect!
     assert_equal '/questions', path

--- a/test/integration/login_flow_test.rb
+++ b/test/integration/login_flow_test.rb
@@ -63,6 +63,16 @@ class LoginFlowTest < ActionDispatch::IntegrationTest
     follow_redirect!
     assert_equal '/questions', path
   end
+  
+  test 'should redirect to dashboard when logging in from /login' do
+    get '/login'
+    assert_response :success
+
+    post '/user_sessions', params: { return_to: request.path, user_session: { username: users(:jeff).username,  password: 'secretive' }  }
+
+    follow_redirect!
+    assert_equal '/dashboard', path
+  end
 
   test 'google login routing' do
     assert_routing '/auth/google_oauth2/callback', {controller: 'user_sessions', action: 'create',provider: 'google_oauth2'}

